nixos/modules/services/networking/charybdis.nix

```diff
@@ -51,46 +51,55 @@ in
         '';
       };
 
+      motd = mkOption {
+        type = types.nullOr types.string;
```

Comment:
`types.string` is deprecated, is using it intentional? Usually, `types.str` or `types.lines` are better choices, depending on what kind of merge semantics you want.